src/django/api/views/auth/user_api_info.py

```diff
@@ -0,0 +1,55 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from api.models import (RequestLog, ApiLimit)
+from ...models import Contributor
+
+
+def get_api_call_limit(contributor_id):
+    try:
+        period_limit = ApiLimit.objects.get(
+            contributor_id=contributor_id).period_limit
+        return str(period_limit)
+    except ApiLimit.DoesNotExist:
+        return '403 Forbidden'
+
+
+def get_renewal_period(contributor_id):
+    try:
+        renewal_period = ApiLimit.objects.get(
+            contributor_id=contributor_id).renewal_period
+        if renewal_period == '':
+            return 'Is not set'
+        else:
+            return renewal_period
+    except ApiLimit.DoesNotExist:
+        return 'Is not set'
+
+
+def get_current_usage(u_id):
+    try:
+        successful_calls = RequestLog.objects.filter(
+            response_code__gte=200,
+            response_code__lte=299,
+            user_id=u_id).count()
+        return str(successful_calls)
+    except RequestLog.DoesNotExist:
+        return 0
+
+
+class UserAPIInfo(APIView):
```

Comment:
Please introduce some verification to ensure that the user who calls this endpoint has the API-related groups set. This will ensure that only authorized and with API groups users can access it. See the example: [api_auth_token](https://github.com/opensupplyhub/open-supply-hub/blob/main/src/django/api/views/auth/api_auth_token.py#L14) file.

Reply:
Good point, I think there will be enough rule for authorized user.

Reply:
done

Reply:
if not request.user.has_groups:
            return Response(status=status.HTTP_403_FORBIDDEN)
          If user doesn't has any group - he/she can't see tab API